tests/test_measurement_model.py

```diff
@@ -0,0 +1,49 @@
+"""Testing the measurement model class."""
+
+import numpy as np
+import polars as pl
+import pytest
+
+from wristpy.core.models import Measurement
+
+
+@pytest.fixture
+def create_mock_measurement() -> Measurement:
```

Comment:
Also this isn't mocking anything; a mock creates a fake interface for something. You're just initializing a required variable.